### ISSUE TITLE
Negative DispIds not supported

### ISSUE THREAD

**Issue by mansellan** [WaynePhillipsEA/twinbasic#318]
**Describe the bug**
Negative DispIds not supported.

**To Reproduce**
```vb
[ DispId (-4)] ' Error: No handler available for this symbol (on the minus sign)
```

**Expected behavior**
Negative DispIds supported.

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting.  I'll get that fixed.   You should be able to use [ Enumerator ] instead

**Comment by Kr00l** [comment]
I tested it already and it worked once. So I assume its a regression.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.3881, thanks for reporting!
